### PR TITLE
[FTAMS102020] Emma Northridge

### PR DESCRIPTION
**Pull request by emmanorthridge**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,13 +6,78 @@
...
+          <p>Stream music on your smartphone, tablet or computer</p>
+        </div>
+        </div>
+
+    </section>
+
+    <section class="green-section">
+       <h2>It's as yeezy as Kanye West.</h2>
+        <div id="green-container">
+       <article class="yeezy">
```
Comment (wax-bear) [inline index.html:62]: The indentation seems off here.

File: index.html [hunk 2]
```diff
@@ -6,13 +6,78 @@
...
+         <div>
+          <h3>Search</h3>
+          <p>Know what you want to listen to? Just search and hit play.</p>
+          <h3>Browse</h3>
+          <p>Check out the latest charts, brand new releases and great playlists for right now.</p>
+          <h3>Discover</h3>
+          <p>Enjoy new music every Monday with your own personal playlist. Or sit back and enjoy Radio.</p>
```
Comment (wax-bear) [inline index.html:69]: The indentation seems off here as well.

File: index.html [hunk 3]
```diff
@@ -6,13 +6,78 @@
...
+      <div class="spotify-header"> 
+      <h2>What's on Spotify?</h2>
+    </div>
+      <div class="info-container">
+        <div class="container">
+          <img class="music-icon" src="images/music-icon.png" alt="music icon" />
+          <h3>Millions of Songs</h3>
+          <p>There are millions of songs on Spotify</p>
+        </div>
+
```
Comment (wax-bear) [inline index.html:43]: The blank lines make the HTML more readable, which is great!

File: styles/style.css [hunk 4]
```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
```
Comment (wax-bear) [inline styles/style.css:6]: Does this comment serve any purpose?

File: styles/style.css [hunk 5]
```diff
@@ -6,3 +6,150 @@ Green: #00B172
...
+
+a {
+    padding: 10px;
+    margin-left: 10px;
+    color: black;
+    font-size: 16px;   
+    text-decoration: none;
+}
+
+
```
Comment (wax-bear) [inline styles/style.css:41]: Please mind unnecessary blank lines.

**Review by wax-bear — COMMENTED** [review]
Great job Emma!

A few comments from my side:
- The UI you submitted closely resembles the original. However, there are a few differences. For example, the nav bar list items seem to be styled differently, and the article text under the h1 title is styled differently as well. Perhaps these minor details could be improved.
- Please mind the indentation and blank lines in the CSS and HTML.

**Comment by emmanorthridge** [comment]
thanks @wax-bear 
Will review the above comments and resubmit!

**Pull request closed without merging**